e2e-tests/IIoTPlatform-E2E-Tests/deploy/IIoTHubDeployment.cs

```diff
@@ -0,0 +1,22 @@
+﻿// ------------------------------------------------------------
+//  Copyright (c) Microsoft Corporation.  All rights reserved.
+//  Licensed under the MIT License (MIT). See License.txt in the repo root for license information.
+// ------------------------------------------------------------
+
+namespace IIoTPlatform_E2E_Tests.TestExtensions {
+    using System.Collections.Generic;
+    using System.Threading.Tasks;
+
+    public interface IIoTHubDeployment {
+
+        /// <summary>
+        /// Create a new layered deployment or update an existing one
+        /// </summary>
+        Task<bool> CreateOrUpdateLayeredDeploymentAsync();
```

Comment:
As an async method interface it should get `CancellationToken`.

Reply:
updated